modules/openxBidAdapter.js

```diff
@@ -187,10 +203,25 @@ const OpenxAdapter = function OpenxAdapter() {
       }
     });
 
-    params.callback = 'window.$$PREBID_GLOBAL$$.oxARJResponse';
-    let queryString = buildQueryStringFromParams(params);
+    if (window.XMLHttpRequest && 'withCredentials' in new XMLHttpRequest()) {
+      try {
+        let queryString = buildQueryStringFromParams(params);
+        let url = `//${delDomain}/w/1.0/arj?${queryString}`;
+        useJsonp = false;
+        ajax.ajax(url, $$PREBID_GLOBAL$$.oxARJResponse, void (0), {
+          withCredentials: true
+        });
+      } catch (err) {
+        useJsonp = true;
+        utils.logMessage(`Ajax call failed due to ${err}, sending ARJ via jsonp instead.`);
+      }
+    }
 
-    adloader.loadScript(`//${delDomain}/w/1.0/arj?${queryString}`);
+    if (useJsonp) {
```

Comment:
When can you not support AJAX?  What I'm trying to say, is in any browser that doesn't support AJAX won't support Prebid.js, so if you can run Prebid.js then you should be able to use AJAX, including `withCredentials` always.